build.sbt

```diff
@@ -112,7 +115,7 @@ val unreleasedModules = Set[String]()
 def youngestForwardCompatible(subProj: String) =
   Some(subProj)
     .filterNot(unreleasedModules.contains)
-    .map { s => "com.twitter" % ("storehaus-" + s + "_2.10") % "0.12.0" }
+    .map { s => "com.twitter" % ("storehaus-" + s + "_2.10") % "0.14.0-RC1" }
```

Comment:
I think we want to drop `_2.10` and let sbt do it: `"com.twitter" %% (s"storehaus-$s") % "0.15.0-RC1"`